frontend/DockerFile

```
@@ -0,0 +1,9 @@
+FROM node:22.13.0 AS build
+WORKDIR /app
+ADD package.json package-lock.json /app
+RUN npm install 
+ADD ./app
```

This does not work
this does:
```suggestion
ADD . /app
```